image_proc/src/resize.cpp

```diff
@@ -107,9 +107,6 @@ void ResizeNode::imageCb(
   sensor_msgs::msg::Image::ConstSharedPtr image_msg,
   sensor_msgs::msg::CameraInfo::ConstSharedPtr info_msg)
 {
-  if (pub_image_.getNumSubscribers() < 1) {
```

Comment:
Why remove this? This can be good to save on traffic if no one's actually listening.

Reply:
As mentioned in the other issue, there's a bug in the getNumSubscribers method that makes it always return 0. I fixed this here, your choice if you'd like me to put it back in

Reply:
Got it, can you please comment it out instead? that will be a placeholder to come back and address this issue.